iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/tck/TieLineTest.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network.impl.tck;
+
+import com.powsybl.iidm.network.tck.AbstractLineTest;
+
+/**
+ * @author Florian Dupuy <florian.dupuy at rte-france.com>
+ */
+class TieLineTest extends AbstractLineTest { }
```

Comment:
This class should extends `AbstractTieLineTest` instead of `AbstractLineTest`:
```suggestion
import com.powsybl.iidm.network.tck.AbstractTieLineTest;

/**
 * @author Florian Dupuy <florian.dupuy at rte-france.com>
 */
class TieLineTest extends AbstractTieLineTest { }
```